shared/haml/onetrust_cookie_scripts.haml

```diff
@@ -0,0 +1,10 @@
+-# OneTrust Cookies Consent Notice scripts for code.org
+- if rack_env?(:production)
+  %script{src: 'https://cdn.cookielaw.org/consent/27cca70a-7db3-4852-9ef0-a6660fd0977d/OtAutoBlock.js', type: 'text/javascript'}
```

Comment:
This link seems to point to a very specific version of the js file. How do we know when to use a newer version?

Reply:
And the js version in `shared/haml/hoc_onetrust_cookie_scripts.haml` is different from the one used here.

Reply:
The hash there isn't locked to a version, it just identifies our script for the *.code.org domains. When we publish a new version of the script via OneTrust, that link will be updated to pull the newest version. It is different from `hoc_onetrust_cookie_scripts` because `*.code.org` and `hourofcode.com` are separate domains, so they unfortunately need separate cookie management. The behavior configuration for both on the OneTrust side is identical though.

Reply:
Got it, thanks for the explanation.